scripts/clean-build-files.sh

```diff
@@ -6,3 +6,29 @@ rm -rf ./docs/builds/sway/latest
 
 # generate sidebar links
 pnpm generate:links
+
+sway_path="docs/sway/docs/book/src"
+fuels_rs_path="docs/fuels-rs/docs/src"
+file_to_rename="README.md"
+
+# rename Sway & Fuels-rs README.md files to index.md
+new_filename="index.md"
```

Comment:
By modifying directly these files, it will generate diff in the git, was happening before, because of that is better to just rename the path/slug after parse

Reply:
the files were just not being parsed though. Currently the wrong README file is shown. 
 This will be fixed in the newer versions as the files were renamed to `index` in the sway and fuels-rs repos.

Reply:
oh it looks like the issue was just related to the config, as it's fixed with the last PR. I will remove this 👍